rand_distr/src/dirichlet.rs

```diff
@@ -46,6 +47,19 @@ pub enum Error {
     SizeTooSmall,
 }
 
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str(match self {
+            Error::AlphaTooShort | Error::SizeTooSmall => {
+                "less than 2 dimensions in Dirichlet distribution"
+            }
+            Error::AlphaTooSmall => "alpha is not positive in Dirichlet distribution",
```

Comment:
Good catch — I guess we should revise these codes during the next breaking release.